templates/tableau-cluster-linux-master.template

```diff
@@ -0,0 +1,706 @@
+                },
+                "Parameters":{  
+                    "AvailabilityZones":{  
+                        "Fn::Join":[  
+                            ",",
+                            {  
+                                "Ref":"AvailabilityZones"
+                            }
+                        ]
+                    },
+                    "KeyPairName":{  
+                        "Ref":"KeyPairName"
+                    },
+                    "NumberOfAZs":"3",
+                    "PrivateSubnet1ACIDR":{  
+                        "Fn::FindInMap":[  
+                            "DefaultConfiguration",
+                            "NetworkConfiguration",
+                            "PrivateSubnet1CIDR"
+                        ]
+                    },
+                    "PrivateSubnet2ACIDR":{  
+                        "Fn::FindInMap":[  
+                            "DefaultConfiguration",
+                            "NetworkConfiguration",
+                            "PrivateSubnet2CIDR"
+                        ]
+                    },
+                    "PrivateSubnet3ACIDR":{  
+                        "Fn::FindInMap":[  
+                            "DefaultConfiguration",
+                            "NetworkConfiguration",
+                            "PrivateSubnet3CIDR"
+                        ]
+                    },
+                    "PublicSubnet1CIDR":{  
+                        "Fn::FindInMap":[  
+                            "DefaultConfiguration",
+                            "NetworkConfiguration",
+                            "PublicSubnet1CIDR"
+                        ]
+                    },
+                    "PublicSubnet2CIDR":{  
+                        "Fn::FindInMap":[  
+                            "DefaultConfiguration",
+                            "NetworkConfiguration",
+                            "PublicSubnet2CIDR"
+                        ]
+                    },
+                    "PublicSubnet3CIDR":{  
+                        "Fn::FindInMap":[  
+                            "DefaultConfiguration",
+                            "NetworkConfiguration",
+                            "PublicSubnet3CIDR"
+                        ]
+                    },
+                    "VPCCIDR":{  
+                        "Fn::FindInMap":[  
+                            "DefaultConfiguration",
+                            "NetworkConfiguration",
+                            "VPCCIDR"
+                        ]
+                    }
+                }
+            }
+        },
+        "BastionHost":{  
+            "Type":"AWS::CloudFormation::Stack",
+            "DependsOn":[  
+                "VPCStack"
+            ],
+            "Properties":{  
+                "TemplateURL":{  
+                    "Fn::Sub": "https://${QSS3BucketName}.s3.amazonaws.com/${QSS3KeyPrefix}templates/linux-bastion.template"
```

Comment:
This should be calling from the submodule itself... the linux-bastion.template that's in there.

Reply:
should it be "https://${QSS3BucketName}.s3.amazonaws.com/${QSS3KeyPrefix}submodules/quickstart-linux-bastion/templates/linux-bastion.template" ?

Reply:
Correct. Plus ensure that the parameters are correct. Please see the example template I put in another comment for more info.